api/src/main/webapp/static/initialize_notebook_runtime.sh

```diff
@@ -48,7 +48,9 @@ cat <<EOF | sudo -E -u jupyter tee ${ignore_file}
 !LICENSE*
 EOF
 
-sudo -E -u jupyter /usr/bin/git config --global core.excludesfile ${ignore_file}
+# Ignore error if this fails. This is a best-effort attempt to set the global gitignore for users.
+sudo -E -u jupyter /usr/bin/git config --global core.excludesfile ${ignore_file} || true
+sudo -E -u jupyter rm -rf ~/.local/share/Trash/files/* || true
```

Comment:
actually it might be 
` /.local/share/Trash/files` without `~`

Reply:
should be /home/jupyter..updated